DNN Platform/Admin Modules/Dnn.Modules.Console/Components/ConsoleController.cs

```diff
@@ -1,6 +1,6 @@
 #region Copyright
 // 
-// DotNetNuke� - http://www.dotnetnuke.com
+// DotNetNuke� https://www.dnnsoftware.comom
```

Comment:
Looks like you have this typo in a number of these files

```suggestion
// DotNetNuke® - https://www.dnnsoftware.com
```